app/javascript/shared/components/tags.jsx

```diff
@@ -1,502 +1,135 @@
-        className={`${classPrefix}__tagoptionrow ${classPrefix}__tagoptionrow--${
-          selectedIndex === index ? 'active' : 'inactive'
-        }`}
-        onClick={this.handleTagClick}
-        onKeyDown={this.handleTagEnter}
-        data-content={tag.name}
-      >
-        <span className={`${classPrefix}__tagname`}>{tag.name}</span>
-        {tag.rules_html && tag.rules_html.length > 0 ? (
-          <button
-            type="button"
-            className={`${classPrefix}__tagsoptionrulesbutton`}
-            onClick={this.handleRulesClick}
-            data-content={tag.name}
-          >
-            {showingRulesForTag === tag.name ? 'Hide Rules' : 'View Rules'}
-          </button>
-        ) : (
-          ''
-        )}
-        {!showingTopTags && (
-          <div
-            className={`${classPrefix}__tagrules--${
-              showingRulesForTag === tag.name ? 'active' : 'inactive'
-            }`}
-            // eslint-disable-next-line react/no-danger
-            dangerouslySetInnerHTML={{ __html: tag.rules_html }}
-          />
-        )}
-      </div>
-    ));
-    if (
-      searchResults.length > 0 &&
-      (activeElement.id === 'tag-input' ||
-        activeElement.classList.contains(
-          'articleform__tagsoptionrulesbutton',
-        ) ||
-        activeElement.classList.contains('articleform__tagoptionrow'))
-    ) {
-      searchResultsHTML = (
-        <div className={`${classPrefix}__tagsoptions`}>
-          {showingTopTags ? (
-            <h2 className={`${classPrefix}__tagsoptionsheading`}>Top tags</h2>
-          ) : null}
-          {searchResultsRows}
-          <div className={`${classPrefix}__tagsoptionsbottomrow`}>
-            Some tags have rules and guidelines determined by community
-            moderators
-          </div>
-        </div>
-      );
-    }
+    return suggestionsResult;
+  };
 
-    return (
-      <div className={`${classPrefix}__tagswrapper crayons-field`}>
-        {listing && (
-          <label htmlFor="Tags" class="crayons-field__label">
-            Tags
-          </label>
-        )}
-        <input
-          data-testid="tag-input"
-          aria-label="Post Tags"
-          id="tag-input"
-          type="text"
-          ref={(t) => {
-            this.textArea = t;
-            return this.textArea;
-          }}
-          className={`${`${fieldClassName} ${classPrefix}`}__tags`}
-          name="listing[tag_list]"
-          placeholder={`Add up to ${maxTags} tags...`}
-          autoComplete="off"
-          value={defaultValue}
-          onInput={this.handleInput}
-          onKeyDown={this.handleKeyDown}
-          onBlur={this.handleFocusChange}
-          onFocus={(e) => {
-            this.fetchTopTagSuggestions();
-            onFocus(e);
-          }}
-          pattern={pattern}
-        />
-        {searchResultsHTML}
-      </div>
-    );
-  }
-}
+  return (
+    <div className={`${classPrefix}__tagswrapper crayons-field`}>
```

Comment:
We don't need this `classPrefix` prop any more - since this component is only being used for Listings now, we can just have 
```suggestion
    <div className="listingform__tagswrapper crayons-field">
```

Reply:
I agree!